components/resource_control/src/resource_group.rs

```diff
@@ -244,6 +245,19 @@ impl ResourceController {
 
         // maybe update existed group
         self.resource_consumptions.write().insert(name, group);
+        self.check_customized();
+    }
+
+    fn check_customized(&self) {
+        let groups = self.resource_consumptions.read();
+        if groups.len() == 1 {
```

Comment:
I think only check `groups.len() == 1` is enough here since when there is only 1 group, there is no difference between fifo and priority queue